full-node/sov-stf-runner/src/config.rs

```diff
@@ -26,12 +25,12 @@ pub struct RpcConfig {
 
 /// Rollup Configuration
 #[derive(Debug, Clone, PartialEq, Deserialize)]
-pub struct RollupConfig<DaServiceConfig> {
-    /// Runner configuration.
+pub struct RollupConfig<DaServiceConfig, StorageConfig> {
+    /// Path to serialized storage configuration.
     pub storage: StorageConfig,
```

Comment:
Would it be possible to use e `PathBuffer` maybe in a netype and remove the `StorageConfig` generic?

Reply:
Yes, it is possible, if we assume that all potential storage implementations will need only path.

Reply:
My inclination would be to make it generic only when we have the use case otherwise we have the generic bubbling up prematurely.